### DISCUSSION TITLE
One of the world’s longest-running mask mandates at 945 days was lifted in early March - where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#291] [Galaxy Trivia] [answered]
**Hint:** 🇭🇰

**Comment by thuongtruong1009** [comment, marked as answer]
The one of the world's longest-running mask mandates at 945 days was lifted in early March 2022 in the city of Hong Kong. The mask mandate was put in place in response to the outbreak of severe acute respiratory syndrome (SARS) in 2003 and continued through the COVID-19 pandemic. The mandate required people to wear masks in most public places, including on public transport, in shopping malls, and at restaurants.
